packages/adsum-map/src/controllers/MainController.js

```diff
@@ -43,6 +43,18 @@ class MainController {
                 this.awm.objectManager.user.setRotation(rotation);
             });
 
+        if (action.zoom) {
+            const { min, max } = action.zoom;
+
+            if (min) {
+                this.awm.cameraManager.control.minDistance = this.awm.getProjector().meterToAdsumDistance(min);
```

Comment:
Add a comment that this is using private AWM api and feature should be added in AWM.

Maybe open an issue on the AWM repo, add add the link here